docs/tutorial/tutorial_cli.ipynb

```diff
@@ -939,7 +939,18 @@
    },
    "source": [
     "### Uploading the CLKs to the entity service\n",
-    "Each party individually uploads its respective CLKs to the entity service. They need to provide the `resource_id`, which identifies the correct results, and an `update_token`."
+    "Each party individually uploads its respective CLKs to the entity service. They need to provide the `resource_id`, which identifies the correct results, and an `update_token`.\n",
+    "\n",
+    "By default, data is uploaded to object store since it has more reliable uploading mechanism especially with large size datasets. You can also specify to upload to entity service directly by adding a flag `--to_entityservice` to command `upload`. A message will be printed to tell users if the encodings and blocks are uploaded to object store:\n",
```

Comment:
... since it has **a** more reliable ... 
... by adding **the** flag ...
.... to **the** command ...